### DISCUSSION TITLE
It is suspected that after using ECOFOW-CLOUD, the user account was stolen and controlled remotely, the author please help

### DISCUSSION THREAD

**Discussion by xiaoxihu2023** [tolwi/hassio-ecoflow-cloud#147] [General]
Strange question (poor English writing, describe concisely):
My ECOFLOW river pro is used as UPS.
Starting from 2023-8-13, the device will automatically turn off the AC switch (this has not happened before).
Checked for a few days and found that very regularly, the AC switch is automatically turned off and then manually turned on. After 9~10 hours, the AC switch will be automatically turned off.
The problem persists after uninstalling ECOFOW-CLOUD, you must register the ECOFLOW app account and re-register the new account to bind the device to eliminate the fault or change the password.
I didn’t use ECOFOW-CLOUD a few days ago, and there is no problem with monitoring the device status with the ECOFLOW app. But after installing ECOFOW-CLOUD in homeassistant, the problem reappeared immediately after configuring the login information.
It is suspected that the user account was stolen and controlled remotely after using ECOFOW-CLOUD.

**Comment by mattwells** [comment]
I have gone through the code that is use for this integration and I have seen not seen anywhere that your login credentials could have been sent outside of your Home Assistant. It is unlikely that there is a security issue with this addon.

I am aware of a security issue with being able to take control of Ecoflow devices about a year ago without having the login details, please make sure that you have updated to the latest firmware on the device. 

It might also be worth resetting the device, on the Delta Pro you hold down the power button for 30 seconds, and then go through the add device procedure again.

**Comment by e1jo** [comment]
if management takes place through the cloud. Try ban internet for ecoflow on your router. It will be visible whether it is a local problem or a global one
